app/lib/components/password_input.svelte

```diff
@@ -0,0 +1,46 @@
+<script>
+  import Flash from 'lib/components/flash.svelte'
+  import { I18n } from 'modules/user/lib/i18n'
+
+  export let password, flash, name
+  let pwdInputType = 'password'
+  let showPassword
+
+  const togglePassword = () => {
+    showPassword = !showPassword
+    if (pwdInputType === 'password') {
+      pwdInputType = 'text'
+    } else {
+      pwdInputType = 'password'
+    }
+  }
+  const changePwd = value => {
+    flash = null
+    password = value
+  }
+</script>
+
+<!--
+Prefer on:change to bind:value since svelte cannot dynamically change a value if another one is binded.
+aka `type={pwdInputType} bind:value={password}` is forbidden since generated code is different for different kinds of input. -->
+<input type={pwdInputType} on:change="{e => changePwd(e.target.value) }" title={name}>
+<Flash bind:state={flash}/>
+<div class="showPasswordWrapper">
+  <input type="checkbox" class="showPassword" id="show{name}" bind:checked={showPassword} on:click={togglePassword}>
```

Comment:
Integration with browser password managers was lost: that regression seems to pre-date this PR, but it would be cool to recover it while we are here

Reply:
addressed in c27dbca